src/domain/checkForAndPlaceOrder.ts

```diff
@@ -442,16 +469,26 @@ async function _placeOrder(params: {
   const { chainId } = orderBook.context;
   try {
     const postOrder: OrderCreation = {
-      ...order,
+      kind: order.kind,
+      from: order.from,
+      sellToken: order.sellToken,
+      buyToken: order.buyToken,
       sellAmount: order.sellAmount.toString(),
       buyAmount: order.buyAmount.toString(),
+      receiver: order.receiver,
       feeAmount: order.feeAmount.toString(),
-      signingScheme: "eip1271",
+      validTo: order.validTo,
+      appData: order.appData,
+      partiallyFillable: order.partiallyFillable,
+      sellTokenBalance: order.sellTokenBalance,
+      buyTokenBalance: order.buyTokenBalance,
+      signingScheme: SigningScheme.EIP1271,
+      signature: order.signature,
```

Comment:
technically not part the PR. It was just annoying me in the logs. The spreading of `order` copy all props, including the `0`, `1`, `2` properties (its a pseudo-array)

With this, logs are less noisy and easier to read